app/_data/install_page.yml

```diff
@@ -0,0 +1,158 @@
+          - icon: /assets/images/icons/third-party/logomark-amazonecs.svg
+            url: /mesh/latest/installation/ecs/
+            name: Amazon ECS
+      - name: Operating Systems
+        cards:
+          - icon: /assets/images/icons/third-party/logomark-centos.svg
+            url: /mesh/latest/installation/centos/
+            name: CentOS
+          - icon: /assets/images/icons/third-party/logomark-redhat.svg
+            url: /mesh/latest/installation/redhat/
+            name: RedHat
+          - icon:
+            url: /mesh/latest/installation/amazonlinux/
+            name: Amazon Linux
+          - icon: /assets/images/icons/third-party/logomark-debian.svg
+            url: /mesh/latest/installation/debian/
+            name: Debian
+          - icon: /assets/images/icons/third-party/logomark-ubuntu.svg
+            url: /mesh/latest/installation/ubuntu/
+            name: Ubuntu
+  gateway_oss:
+    icon: /assets/images/logos/kong-gateway-logo.svg
+    description: Grains lemonade zest cilantro cozy cinnamon oatmeal comforting pumpkin spice latte.
+    items:
+      - name: Supported by Kong
+        cards:
+          - icon: /assets/images/icons/third-party/logomark-docker.svg
+            url: /gateway/latest/install/docker/?install=oss
+            name: Docker
+          - icon:
+            url: /gateway/latest/install/linux/amazon-linux/?install=oss
+            name: Amazon Linux
+          - icon: /assets/images/icons/third-party/logomark-kubernetes.svg
+            url: /kubernetes-ingress-controller/latest/deployment/k4k8s/
+            name: Kubernetes
+          - icon: /assets/images/icons/third-party/logomark-debian.svg
+            url: /gateway/latest/install/linux/debian/?install=oss
+            name: Debian
+          - icon: /assets/images/icons/third-party/logomark-redhat.svg
+            url: /gateway/latest/install/linux/rhel/?install=oss
+            name: RedHat
+          - icon: /assets/images/logos/kong-kuma.png
+            url: /mesh/latest/explore/gateway/
+            name: Kong Gateway on Kuma
+          - icon: /assets/images/logos/kong-istio.png
+            url: /kubernetes-ingress-controller/latest/guides/getting-started-istio/
+            name: Kong Gateway on Istio
+          - icon: /assets/images/icons/third-party/logomark-ubuntu.svg
+            url: /gateway/latest/install/linux/ubuntu/?install=oss
+            name: Ubuntu
+      - name: Supported by the Community
+        cards:
+          - icon: /assets/images/icons/third-party/logomark-azure.svg
+            url: /hub/bitnami/microsoft_azure?install=oss
+            name: Kong on Microsoft Azure
+          - icon: /assets/images/icons/third-party/logomark-terraform.svg
+            url: /hub/kong-inc/kong-terraform-aws/
+            name: Kong on AWS with Terraform
+  kuma:
+    icon: /assets/images/logos/kuma-logo.svg
+    description: Grains lemonade zest cilantro cozy cinnamon oatmeal comforting pumpkin spice latte.
```

Comment:
```suggestion
    description: Built for hybrid and multi-cloud environments, optimized for microservices and distributed architectures. If you want to run Kuma as a part of a production-ready API platform, you can [get started with Konnect] (https://konghq.com/products/kong-konnect/register?utm_medium=referral&utm_source=docs&utm_campaign=gateway-konnect&utm_content=docker-install) in under 5 minutes.
```

Reply:
Replacing placeholder text w/ reused copy from original /install/ page + @connorckong's suggested CTA from March 4th discussion & comment in JIRA.

Need to confirm the CTA URL, currently using link from /gateway/latest/install/docker/#prerequisites

Reply:
@fabianrbz , you can disregard these suggestions, they'll be obsoleted by the new install plans coming from Connor.